docs/src/components/VersionDropdown.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useEffect } from 'react';
+
+const CACHE_KEY = 'kargo-docs-versions';
+const CACHE_DURATION = 1000 * 60 * 30; // 30 minutes
+const githubApiUrl = 'https://api.github.com/repos/akuity/kargo/branches';
```

Comment:
```suggestion
const githubApiUrl = 'https://api.github.com/repos/akuity/kargo/branches?protected=true';
```

would piggyback on the fact that we use branch protection for all our release branches, while reducing the size of the payload.